store/src/java/com/zimbra/cs/account/soap/SoapProvisioning.java

```diff
@@ -1235,19 +1238,59 @@ public long getNumRemaining() {
             Progress() {
             }
 
-            Progress(long succeeded, long failed, long remaining) {
+            public int getStatusCode() {
+                return statusCode;
+            }
+
+            Progress(int status, long succeeded, long failed, long remaining, String account) {
+                statusCode = status;
```

Comment:
have you checked if the changes to re-indexing API break reindexing UI in Admin Console? I recall having to make changes to the UI code, but I may be mistaken. If these changes break the UI, please file a ticket for followup UI work.

Reply:
I will check this.

Reply:
As far as I can tell, re-indexing accounts works as before. There will need to be UI work for server-level re-indexing though.